blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubScmContent.java

```diff
@@ -0,0 +1,84 @@
+package io.jenkins.blueocean.blueocean_github_pipeline;
+
+import io.jenkins.blueocean.commons.ErrorMessage;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Vivek Pandey
+ */
+public class GithubScmContent {
```

Comment:
Seems like this could be generic, too... similarly with `payload` instead of `sha`